src/main/java/mezz/jei/gui/ghost/GhostIngredientDragManager.java

```diff
@@ -122,7 +122,28 @@ public void stopDrag() {
 	}
 
 	private <T extends Screen, V> boolean handleClickGhostIngredient(T currentScreen, IClickedIngredient<V> clicked, double mouseX, double mouseY) {
-		IGhostIngredientHandler<T> handler = guiScreenHelper.getGhostIngredientHandler(currentScreen);
+		IGhostIngredientHandler<T> handler = // guiScreenHelper.getGhostIngredientHandler(currentScreen);
```

Comment:
what's all this?

Reply:
Oh shit... I didn't intend to commit that 😁 
That's what I get for making that line-width change half-asleep and forgetting to exclude my "test" code.
I will remove it and re-request review....